packages/notifi-react-card/lib/components/defaults.css

```diff
@@ -16,17 +14,25 @@
   --notifi-alert-color: rgb(252 0 0);
   --notifi-link-color: #0000ee;
   --notifi-checkmark-color: #22af3c;
+  --notifi-intercom-card-background: #fff;
+  --notifi-intercom-font-color: #262949;
+  --notifi-intercom-secondary-font-color: #80829d;
+  --notifi-intercom-intro-color: #262949;
+  --notifi-intercom-input-background: #f5f6fb;
+  --notifi-intercom-input-border: #e8ebf5;
+  --notifi-intercom-button-color: #1448f3;
+  --notifi-intercom-button-disabled-color: #b6b8d5;
+  --notifi-intercom-incoming-message: #f5f6fb;
+  --notifi-intercom-outgoing-message: #e5ebff;
```

Comment:
actually, can you consolidate these colors with the subscription card and just use one variable name for both subs and intercom?

Feel free to change look & feel of the default subscription card.

Reply:
okok will do

Reply:
updated!

Reply:
Hey @juni-notifi , could you re-review when you get a chance? Merging is blocked